Resources/Prototypes/Recipes/Reactions/drinks.yml

```diff
@@ -1117,3 +1117,13 @@
       amount: 1
   products:
     Cola: 4
+
+- type: reaction
+  id: Coffee
+  reactants:
+    Water:
+      amount: 3
+    CoffeeGrounds:
+      amount: 1
+  products:
+    Coffee: 4
```

Comment:
Consider adding a minimum temperature this reaction occurs at.  Mmm, microwave coffee.

Reply:
This one I left out on purpose as water has a boiling point. I have no clue how it would process it if water want too high too fast. Would it process or boil off? no clue! :)

Reply:
Might be wrong, but off the top of my head, the entire solution processes at once at the moment.  If you had a beaker with 40u of water, 10u of coffee grounds, and had a minimum temperature here of, say, 80 degrees C.

You put the beaker in the microwave, set the timer for 30 seconds (for argument's sake).
Until the beaker heats up to 80 degrees, nothing happens.  When it hits 80 degrees, the solution changes to 40u coffee and 10u water.  The microwave then gets up to 100 C, but I don't believe the water boils off.

You can test what happens to water by setting the temperature of a solution in a beaker in a sandbox environment.  I don't think it boils, but I'm not _positive_.